### PR TITLE
[Feature] Triqler output with decoys

### PR DESCRIPTION
**Pull request by jpfeuffer**
Adds decoys when triqler is enabled.

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] `CHANGELOG.md` is updated
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] Documentation in `docs` is updated
- [x] If necessary, also make a PR on the [nf-core/proteomicslfq branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/proteomicslfq)

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -1110,7 +1113,8 @@ process proteomicslfq {
                    -out out.mzTab \\
                    -threads ${task.cpus} \\
                    ${msstats_present} \\
-		   ${triqler_present} \\
+                   ${triqler_present} \\
+                   ${decoys_present} \\
```
Comment (ypriverol) [inline main.nf:1117]: @jpfeuffer do you think the mzTab is exporting by default also the decoys?

Comment (jpfeuffer) [reply main.nf:1117]: Yes, if decoys are activated, the mzTab should have them too.

Comment (jpfeuffer) [reply main.nf:1117]: You can check the GitHub Actions artifacts to see how the output files look like.

**Comment by ypriverol** [comment]
You can merge the PR @jpfeuffer

**Pull request merged**